### ISSUE TITLE
[Bug] newest gifski does not work with screengif

### ISSUE THREAD

**Issue by Herr-Sepp** [NickeManarin/ScreenToGif#1214]
**Describe the bug**
Using the newest gifski dll creates broken gif files

**To Reproduce**
1. download newest gifski .dll from https://gif.ski/gifski-1.11.0.zip
2. save as ... with gifski
3. quality settings do not matter
4. the gif is always 8,00 KB or 4,00 KB and broken

**Expected behavior**
the .gif works

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 2.38.1

**Comment by cherinyy** [comment]
I noticed that they provide a shared library in [1.5.0](https://gif.ski/gifski-1.5.0.zip), but ScreenToGif doesn't work with it. This means that ScreenToGif does not use the officially built Gifski shared library.

This issue should probably be closed and a feature request issue raised to allow the use of the official gifski shared library.

**Comment by NickeManarin** [comment]
I tested with that version and it works normally btw.

**Comment by byzod** [comment]
> I tested with that version and it works normally btw.

How? What version are you talking about? I downloaded both https://gif.ski/gifski-1.11.0.zip and https://gif.ski/gifski-1.5.0.zip mentioned above but none of them working

**Comment by CosmicGrill** [comment]
It seems that the rust code rewrite made latest gifski DLL libraries (taken from the win/developer folder of the tar.gz archive) work again with S2G, albeit not properly:

1.14.1 and 1.14.4 have both problems with GIFs being sped-up (as if the new encoder build didn't respect frame delays) and an unexpectionally long pause at the end of the gif playback before it starts looping again.
BTW: Does the "loop forever" option even work? It feels like the output gif still loops in my browser & irfanview when that option is disabled. Tested with the bundled-as-plugin DLL and with 1.14.4

An 1.14.1-specific issue is that it has visual glitches in the exported gif, but not sure if that's just gifski-specific. Regardless, 1.14.4 doesn't exhibit this problem.

Not sure of any other issues existing that haven't exhibited in the bundled-as-plugin library from 2020-09.

I attached the project file that I used to test gifski onto.
Imported the gifski-1.14.4 output gif to a new project - all but the last frames are set to a 20ms delay, with the last frame being 16090 ms. Imported the output gif that was exported with the default gifski version (I believe it was 1.2.0, yeah?) to a new project, and the timings were intact.

Surprised to see the bundled gifski build working with variable frame delays, as seeing gifski's issue tracker - it's not really supported.

[test_project.stg.zip](https://github.com/NickeManarin/ScreenToGif/files/14860279/test_project.stg.zip)

**Comment by Putarku** [comment]
By the end of 2024, I revisited this issue, but unfortunately, it remains unresolved. Gifski has now been updated to version 1.32.0, yet when used in conjunction with S2G, it still produces erroneous GIF outputs.

Currently, S2G can only utilize the outdated version that is downloaded by default, which has poor compression quality, resulting in excessive noise and color blocks. This is significantly inferior to the newer versions of Gifski available today. 

Therefore, I hope the author can address this issue. Thank you : )
